statvar_imports/us_college_ipeds/preprocess.py

```diff
@@ -0,0 +1,36 @@
+import os
```

Comment:
Can you add a top level description to this file which you already documented in the README file please?

```
"""
Cleans the raw CSV files by removing initial descriptive rows.
"""
```